src/client/src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { shallow } from "enzyme";
+import { Card } from "./index";
+
+describe("Card", () => {
```

Comment:
Similar thing here. You can probably write the description as "Card Component renders appropriately". In fact, in future, when we add more tests we can be more precise on what are we trying to test with this describe statement.

Reply:
Like above. IMO it's good :)